ledger/src/blockstore_db.rs

```diff
@@ -469,10 +464,10 @@ impl Rocks {
     /// software version that adds a new column, and then also opened with an older version that
     /// did not have knowledge of that new column.
     fn cf_descriptors(
+        path: &Path,
         options: &BlockstoreOptions,
         oldest_slot: &OldestSlot,
-        detected_cfs: &[String],
-    ) -> Vec<ColumnFamilyDescriptor> {
+    ) -> Result<Vec<ColumnFamilyDescriptor>> {
```

Comment:
Will this ever error? If not, probably revert this change.

Reply:
Nope - think I just confused myself when digging into that rocksdb function; reverted